api/python/t4/data_transfer.py

```diff
@@ -621,6 +624,19 @@ def _process_url(src):
                         hash_obj.update(chunk)
                         with lock:
                             progress.update(len(chunk))
+
+                    current_file_size = fd.tell()
+                    if current_file_size != size:
+                        old_size = humanize.naturalsize(size)
```

Comment:
And actually, I don't think `naturalsize` is a good idea... E.g.:
```
In [11]: humanize.naturalsize(10000)                                                                                                                                                                                            
Out[11]: '10.0 kB'

In [12]: humanize.naturalsize(10010)                                                                                                                                                                                            
Out[12]: '10.0 kB'
```

(This is a rare case, so being precise is more useful than being user-friendly)

Reply:
Done.